data/database/src/commonMain/kotlin/com/tunjid/heron/data/database/entities/LabelDefinitionEntity.kt

```diff
@@ -30,3 +33,12 @@ data class LabelDefinitionEntity(
     val severity: String,
     val localeInfoCbor: String,
 )
+
+fun LabelDefinitionEntity.asExternalModelWithLocale(): Pair<Label.Definition, Labeler.LocaleInfoList> =
```

Comment:
We don't seem to use the value of `Pair.second` when this method is called.

I think we should just include the `LocaleList` in the `Label.Definition` class. Foe the default one we use, it can just be an empty LocaleList. he next PR will will be replacing it anyway.